lib/elixir_console_web/live/console_live/command_input_component.html.leex

```diff
@@ -1,6 +1,8 @@
+<% alias ElixirConsoleWeb.ConsoleLive %>
```

Comment:
I would remove this, and put the following in the `CommandInputComponent` module:

`import ElixirConsoleWeb.ConsoleLive, only: [print_prompt: 0]`

(or simply `import ElixirConsoleWeb.ConsoleLive.Helpers` if you buy the dedicated module for helpers that I mentioned before).

So we can simply invoke `print_prompt()`. This kind of helper functions to be used in templates seems to be a good use of `import`, following how `Phoenix.LiveView.Helpers` are imported in the `lib/elixir_console_web.ex`.

Reply:
sounds good, i'll do that creating the helpers module which sounds appropriate. Thanks for the suggestions